src/functions.jl

```diff
@@ -109,3 +109,9 @@ for func in wgslfunctions
 	end |> eval
 end
 
+# Bitcast is internal function not exported
```

Comment:
bitcast function is a wgsl function that needs to be included in the future. Keeping it here for reference.